README.md

```diff
@@ -213,32 +223,27 @@ Finally, we use `host` `network_mode`, this is because our server code refers to
 
 </details>
 
-## Start
-
-Make sure you had built the image via `bash build_image.sh`
-Start all services at once with the following command, however it may clog up the terminal window as they all run in the same terminal so you may run some services in detached mode.
-
-`docker compose up`
+## Quick Start
 
-To start multiple containers on a machine, use the following command
+We require our Params FTP Server to be running before starting the prover node. The prover node must copy the parameters from the FTP server to it's own volume to operate correctly.
 
-`docker compose -p <node> up` where `node` is the unique name of the container/project you would like to start.
+### Params FTP Server
 
-Ensure the docker compose file has GPU's specified for each container.
+Start the FTP server with `docker compose up params-ftp -f ftp-docker-compose.yml`.
 
-### Starting individual services
+The default port is `21` and the default user is `ftpuser` with password `ftppassword`. The ports used for file transfer are `30000-30009`.
 
-It may be cleaner to start services individually. You can start each in a new terminal window, or in the background.
+### Prover Node
 
-To start each service in the background, use the following command
+Make sure you had built the image via `bash build_image.sh`
 
-`docker compose start <service>`
+Once the Params FTP server is running, you can start the prover node.
 
-To start an attached service, use the following command:
+Start all services at once with the following command, however it may clog up the terminal window as they all run in the same terminal so you may run some services in detached mode.
 
-`docker compose up <service>`
+`docker compose up prover-node` This will run the base services in order of mongodb, dry-run-service, prover-node
```

Comment:
This command start the 3 container service but if we ctrl-c it , only prover node container stop.
So we can think about add a "stop service" section to let user know how to stop all services started if they want to stop all of them